sharktank/sharktank/models/llama/llama.py

```diff
@@ -27,7 +27,7 @@
 ################################################################################
 
 
-class PagedLlamaModelV1(BaseCausalLMModel):
+class PagedLlamaModelV1(BaseCausalLMModel, CausalLMModel):
```

Comment:
I guess the naming is not good. One of them is implementation and the other is an ABC. I will rename them to something more clear. I did it like that because BaseCausalLMModel implements just a part of the whole CausalLMModel interface.